tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs

```diff
@@ -23,7 +23,7 @@ public NodeAngularAppTest(ITestOutputHelper output, TestTempDirTestFixture fixtu
         public const int PortInContainer = 4200;
 
         // Official Node.js version that is supported by Angular CLI 6.0+ is 8.9 or greater
-        [Theory]
+        [Theory(Skip = "Legacy node versions are out of support")]
```

Comment:
Do we plan on un-skipping? If not, should we remove the legacy tests altogether?

Reply:
I don't think there's any chance we'll unskip these tests in the future, so yes I can remove these tests.